src/templates/nssfcfg.conf.j2

```diff
@@ -0,0 +1,88 @@
+configuration:
+  nrfUri: {{ nrf_url }}
+  nsiList:
+  - nsiInformationList:
+	- nrfId: {{ nrf_url }}/nnrf-nfm/v1/nf-instances
+	  nsiId: 22
+	snssai:
+	  sd: "010203"
```

Comment:
Those would be provided by simapp/webui so if possible I'd remove them from here. It makes it confusing reading at config files if we see values that aren't really used.

Ref [here](https://github.com/gruyaume/omec-simapp-operator/blob/main/src/files/default_config.yaml#L89)

Reply:
I removed the `snssai` section.